### DISCUSSION TITLE
code_aster can't load behaviour for hypothesis 'Tridimensional'

### DISCUSSION THREAD

**Discussion by mora1n** [thelfer/tfel#602] [Q&A] [answered]
Hello @thelfer, I test `my_behaviour.mfront` in 2 different versions of code_aster. The old version of code_aster with mfront-3.4.3 compiled sucessfully `libAsterBehaviour.so` and finished the calculation without any problem. The new version code_aster with mfront-4.1.0 also complied sucessfully but threw out errors as follows when calculating:
`LibrariesManager::getBehaviour: can't load behaviour 'astermybehaviour' in library 'fort.21' for hypothesis 'Tridimensional'` (where fort.21 is the unite of `libAsterBehaviour.so`)

**Comment by mora1n** [comment, marked as answer]
I thought I've found the solution. I tried using macro command `CREA_LIB_MFRONT` in code_aster and I found that the `libBehaviour.so` is the same as I compiled mannully with `--interface=generic`. Is that means in the future we don't need `--interface=aster` for using mfront in code_aster?

**Reply by maxence-wz** [reply]
@Mora1n, exactly ! ```code_aster``` will natively use the generic MFront interface.
